test/framework/resources/templates/vpntunnel.yaml.tpl

```diff
@@ -27,7 +27,7 @@ spec:
               sleep 3600;
             done
       - name: logger
-        image: registry.k8s.io/e2e-test-images/agnhost:2.40
+        image: eu.gcr.io/gardener-project/3rd/agnhost:2.40
```

Comment:
The vpntunnel Deployment is deployed in the Shoot.
https://github.com/gardener/gardener/blob/2f34cb1422985253aa47eb761bb759d6ae4823e3/test/testmachinery/shoots/vpntunnel/vpntunnel.go#L68-L69

Isn't better to leave it in the upstream version?

Reply:
I changed it for consistency, but indeed it is not needed. I will revert this change.